backend/LeetWars.Core/LeetWars.Core.DAL/Entities/ChallengeLevel.cs

```diff
@@ -4,14 +4,14 @@ namespace LeetWars.Core.DAL.Entities
 {
     public class ChallengeLevel : Entity<long>
     {
-        public string Name { get; set; }
+        public SkillLevel DifficultyLevel { get; set; }
```

Comment:
we don't need Language level and skill level, please leave only one of them (probably this one)

Reply:
As I investigated, it would be quite problematic considering that we have a good number of functions depending on that LanguageLevel property as well as a separate column in a database fully dedicated to it